core/jobs/batch_jobs/collection_info_jobs_test.py

```diff
@@ -25,27 +25,31 @@
 from core.jobs.types import job_run_result
 from core.platform import models
 
+from typing import Type
+from typing_extensions import Final
+
 MYPY = False
 if MYPY: # pragma: no cover
     from mypy_imports import collection_models
     from mypy_imports import feedback_models
     from mypy_imports import user_models
 
-(
-    collection_models, feedback_models, user_models
-) = models.Registry.import_models([
-    models.NAMES.collection, models.NAMES.feedback, models.NAMES.user])
-
-datastore_services = models.Registry.import_datastore_services()
+(collection_models, feedback_models, user_models) = (
+    models.Registry.import_models([
+        models.NAMES.collection, models.NAMES.feedback, models.NAMES.user
+    ])
+)
 
 
 class GetCollectionOwnersEmailsJobTests(job_test_utils.JobTestBase):
 
-    JOB_CLASS = collection_info_jobs.GetCollectionOwnersEmailsJob
+    JOB_CLASS: Type[
+        collection_info_jobs.GetCollectionOwnersEmailsJob
+    ] = collection_info_jobs.GetCollectionOwnersEmailsJob
```

Comment:
Wouldn't just `Final` work here?

Reply:
ditto elsewhere

Reply:
`Final ` only work for read-only values ( constants ), but here `JOB_CLASS` is reassigned because `JOB_CLASS` is already defined in super class.

Reply:
I see